internal/openshift/whoami.go

```diff
@@ -0,0 +1,56 @@
+package openshift
```

Comment:
Why isn't this just a method of OpenShift Client?

Reply:
Good point. I might move it there. This is actually part of the code I took from Tenant. There things are bit differently structured. But I see where you are coming from. I have a look, if I can easily move it into our OpenShiftClient.

Reply:
Done. I added the whoami call to the existing OpenShiftClient API. Thanks